build/api-server/Dockerfile

```diff
@@ -9,8 +9,18 @@ ARG VERSION
 ARG COMMIT 
 ARG ANALYTICS_API_KEY 
 ARG ANALYTICS_TRACKING_ID
+ARG SLACK_BOT_CLIENT_ID 
+ARG SLACK_BOT_CLIENT_SECRET
 
-RUN cd cmd/api-server;go build -ldflags "-X github.com/kubeshop/testkube/internal/pkg/api.Version=$VERSION -X github.com/kubeshop/testkube/internal/pkg/api.Commit=$COMMIT -X github.com/kubeshop/testkube/pkg/analytics.TestkubeMeasurementID=$ANALYTICS_TRACKING_ID -X github.com/kubeshop/testkube/pkg/analytics.TestkubeMeasurementSecret=$ANALYTICS_API_KEY" -o /app -mod mod -a .
+RUN cd cmd/api-server; \
+    go build \
+        -ldflags "-X github.com/kubeshop/testkube/internal/pkg/api.Version=$VERSION \
+            -X github.com/kubeshop/testkube/internal/pkg/api.Commit=$COMMIT \
+            -X github.com/kubeshop/testkube/pkg/analytics.TestkubeMeasurementID=$ANALYTICS_TRACKING_ID \
+            -X github.com/kubeshop/testkube/pkg/analytics.TestkubeMeasurementSecret=$ANALYTICS_API_KEY \
+			-X github.com/kubeshop/testkube/pkg/slacknotifier.SlackBotClientID=$SLACK_BOT_CLIENT_ID \
```

Comment:
Do we really need to embed slack into binary? It need to change for each client?

Reply:
we need only one instance because slack will access it to validate the bot, it is not needed for every instance.